packages/vite/src/node/ssr/inlineImport.ts

```diff
@@ -0,0 +1,70 @@
+import type { InlineConfig } from '../config'
+import { resolveConfig } from '../config'
+import { createRunnableDevEnvironment } from '../server/environments/runnableEnvironment'
+import { mergeConfig } from '../utils'
+
+interface InlineImportResult<T> {
+  module: T
+  dependencies: string[]
+}
+
+/**
+ * Import any file using the default Vite environment.
+ * @experimental
+ */
+export async function inlineImport<T>(
+  moduleId: string,
+  inlineConfig?: InlineConfig,
+): Promise<InlineImportResult<T>> {
+  const environment = createRunnableDevEnvironment(
+    'inline',
+    // TODO: provide a dummy config?
+    await resolveConfig(
+      mergeConfig(inlineConfig || {}, {
+        environments: {
+          inline: {
+            consumer: 'server',
+            dev: {
+              moduleRunnerTransform: true,
+            },
+            resolve: {
```

Comment:
Should `module`/`development`/`production` condition and `module` field be used here? In the bundled config loader, we don't use them to align with node.

Reply:
Ok, I just copied it from the bundleConfig now